examples/examples/data-channels-flow-control/data-channels-flow-control.rs

```diff
@@ -1,368 +1,263 @@
+        let more_can_be_sent = more_can_be_sent.clone();
+
         Box::pin(async move {
-            let _ = send_more_ch_tx2.send(()).await;
+            more_can_be_sent.send(()).await.unwrap();
         })
     }))
     .await;
 
-    Ok(pc)
+    pc
 }
 
-async fn create_answerer() -> Result<Arc<RTCPeerConnection>> {
-    // Create a MediaEngine object to configure the supported codec
-    let mut m = MediaEngine::default();
-
-    // Register default codecs
-    m.register_default_codecs()?;
+async fn create_responder() -> RTCPeerConnection {
+    let pc = create_peer_connection().await;
 
-    // Create a InterceptorRegistry. This is the user configurable RTP/RTCP Pipeline.
-    // This provides NACKs, RTCP Reports and other features. If you use `webrtc.NewPeerConnection`
-    // this is enabled by default. If you are manually managing You MUST create a InterceptorRegistry
-    // for each PeerConnection.
-    let mut registry = Registry::new();
-
-    // Use the default set of Interceptors
-    registry = register_default_interceptors(registry, &mut m)?;
+    pc.on_data_channel(Box::new(move |dc| {
+        Box::pin(async move {
+            let total_bytes_received = Arc::new(AtomicUsize::new(0));
 
-    // Create the API object with the MediaEngine
-    let api = APIBuilder::new()
-        .with_media_engine(m)
-        .with_interceptor_registry(registry)
-        .build();
+            let get_total_bytes_received = total_bytes_received.clone();
+            dc.on_open(Box::new(move || {
+                Box::pin(async {
+                    println!("responder :: on_open");
 
-    // Prepare the configuration
-    let config = RTCConfiguration {
-        ice_servers: vec![RTCIceServer {
-            //urls: vec!["stun:stun.l.google.com:19302".to_owned()],
-            ..Default::default()
-        }],
-        ..Default::default()
-    };
+                    let task = tokio::spawn(async move {
+                        let start = SystemTime::now();
 
-    // Create a new RTCPeerConnection
-    let pc = Arc::new(api.new_peer_connection(config).await?);
+                        tokio::time::sleep(Duration::from_secs(1)).await;
+                        println!("");
 
-    pc.on_data_channel(Box::new(move |dc: Arc<RTCDataChannel>| {
-        let total_bytes_received = Arc::new(AtomicUsize::new(0));
+                        loop {
+                            let total_bytes = get_total_bytes_received.load(Ordering::Relaxed);
 
-        Box::pin(async move {
-            // Register channel opening handling
-            let total_bytes_received2 = Arc::clone(&total_bytes_received);
-            dc.on_open(Box::new(move || {
-                println!("OnOpen: Start receiving data"); //, dc2.label(), dc2.id());
-                tokio::spawn(async move {
-                    let since = SystemTime::now();
+                            let elapsed = SystemTime::now().duration_since(start);
```

Comment:
We should switch to using `Instant` instead of `SystemTime` to measure elapsed time. Doesn't have to happen in this PR, however